packages/strapi-design-system/src/Select/Select.js

```diff
@@ -169,12 +169,14 @@ export const Select = ({
             ref={buttonRef}
             labelledBy={`${generatedId} ${labelId} ${contentId}`}
             aria-describedby={ariaDescribedBy}
+            aria-required={required}
             expanded={Boolean(expanded)}
             onTrigger={setExpanded}
             id={generatedId}
             hasError={Boolean(error)}
             disabled={disabled}
             onMouseDown={handleMouseDown}
+            value={multi ? value.join(',') : value}
```

Comment:
Because of how this select is built it has no value, the value is rendered in a random div layered on top of it, the combobox needs some kind of value.